react-components/src/data/search/SearchForm.js

```diff
@@ -0,0 +1,374 @@
+                                {dateIntervalChildren}
+                            </Field>
+                        </td>
+                    </tr>
+
+                    <tr>
+                        <td>
+                            <Field name='negatedFileQuery'
+                                   id={ids.nameHasNot}
+                                   label={getMessage('nameHasNot')}
+                                   component={renderTextField}/>
+                        </td>
+                        <td>
+                            <Field name='modifiedWithin'
+                                   id={ids.modifiedWithin}
+                                   label={getMessage('modifiedWithin')}
+                                   component={renderDropDown}>
+                                {dateIntervalChildren}
+                            </Field>
+                        </td>
+                    </tr>
+
+                    <tr>
+                        <td>
+                            <Field name='metadataAttributeQuery'
+                                   id={ids.metadataAttributeHas}
+                                   label={getMessage('metadataAttributeHas')}
+                                   component={renderTextField}/>
+                        </td>
+                        <td>
+                            <Field name='ownedBy'
+                                   id={ids.ownedBy}
+                                   label={getMessage('ownedBy')}
+                                   placeholder={intl.formatMessage({id: 'enterCyVerseUserName'})}
+                                   component={renderTextField}/>
+                        </td>
+                    </tr>
+
+                    <tr>
+                        <td>
+                            <Field name='metadataValueQuery'
+                                   id={ids.metadataValueHas}
+                                   label={getMessage('metadataValueHas')}
+                                   component={renderTextField}/>
+                        </td>
+                        <td>
+                            <Field name='sharedWith'
+                                   id={ids.sharedWith}
+                                   label={getMessage('sharedWith')}
+                                   placeholder={intl.formatMessage({id: 'enterCyVerseUserName'})}
+                                   component={renderTextField}/>
+                        </td>
+                    </tr>
+
+                    <tr>
+                        <td>
+                            <table>
+                                <tbody>
+                                <tr>
+                                    <td className={classes.fullWidth}>
+                                        <Field name='fileSizeRange.min'
+                                               type='number'
+                                               parse={value => value ? Number(value) : null}
```

Comment:
It works, even with 0.  I initially had `value => Number(value)`, but then if you touched the input field at all it would automatically assign 0 as the value with no way of removing it 😖